api/http/management.go

```diff
@@ -27,3 +37,66 @@ func NewManagementAPI(app app.App) *ManagementAPI {
 		App: app,
 	}
 }
+
+func (api *ManagementAPI) ConfigurationSet(c *gin.Context) {
```

Comment:
Not super important, but I would suggest VerbNoun naming, thus `SetConfiguration`

Reply:
changed.